react/Spinner/Readme.md

```diff
@@ -0,0 +1,49 @@
+Show a spinner when loading something.
+
+### Default 
+
+```
+<Spinner />
+```
+
+### Color
+
+```
+<div>
+  blue: <Spinner color='blue' />
+  grey: <Spinner color='grey' />
+  white: <Spinner color='white' />
+  red: <Spinner color='red' />
+</div>
+```
+
+### Placement
+
+```
+<div>
+<Spinner noMargin={ true }/>
+</div>
+```
+
+### Sizes
+
+```
+<div>
+  tiny: <Spinner size='tiny' /><br/>
+  small: <Spinner size='small' /><br/>
+  medium: <Spinner size='medium' /><br/>
+  large: <Spinner size='large' /><br/>
+  xlarge: <Spinner size='xlarge' /><br/>
+  xxlarge: <Spinner size='xxlarge' />
+</div>
+```
```

Comment:
`tiny`, `small` and `medium` are the exact same size, see:

![](https://i.imgur.com/FnehSpi.png)

Reply:
Bug from the spinner

Reply:
https://github.com/cozy/cozy-ui/issues/197